muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/records/body/application/records.tsx

```diff
@@ -206,11 +206,11 @@ class Records extends React.Component<RecordsProps, RecordsState> {
           let records = data.records;      
 
           return <div className="react-required-container" key={data.user.id}>
-          <div className="application-sub-panel__header text text--studies-header">{user.studyProgrammeName}</div>
+          <div className="application-sub-panel__header">{user.studyProgrammeName}</div>
           <div className="application-sub-panel__body">
             {records.length ? records.map((record, index)=>{
               return <ApplicationList key={record.groupCurriculumIdentifier || index}>
-                {record.groupCurriculumIdentifier ? <div className="application-list__header"><h3 className="text text--studies-list-header">{storedCurriculumIndex[record.groupCurriculumIdentifier]}</h3></div> : null}  
+                {record.groupCurriculumIdentifier ? <div className="application-list__header-container"><h3 className="application-list__header">{storedCurriculumIndex[record.groupCurriculumIdentifier]}</h3></div> : null}  
```

Comment:
Shouldn't the h3 elements className be application-list__header-title as used previously?

If we have application-list__header-container element then we should have some child node as application-list__header-title where only child element is text node. At least previously className application-list__header-title has been set to an element where only child node is text node.